### PR TITLE
Add MapType: Google Hybrid

### PR DESCRIPTION
**Pull request by ans10528**
image
image

### CODE REVIEW

**Comment by DonLakeFlyer** [comment]
@tilaktilak Do you know enough about the mapping stuff now to verify this is correct?

File: src/Settings/FlightMapSettings.cc [hunk 1]
```diff
@@ -86,7 +86,7 @@ void FlightMapSettings::_newMapProvider(QVariant value)
         _removeEnumValue(mapTypeTerrain, enumStrings, enumValues);
         break;
     case mapProviderGoogle:
-        _removeEnumValue(mapTypeHybrid, enumStrings, enumValues);
+        //_removeEnumValue(mapTypeHybrid, enumStrings, enumValues);
```
Comment (DonLakeFlyer) [inline src/Settings/FlightMapSettings.cc]: Why is this commented out instead of having the entire case removed?

**Comment by DonLakeFlyer** [comment]
What changed to make this possible now where it wasn't possible before?

**Comment by tilaktilak** [comment]
ans10528 add a new argument that was missing in the URL (`_versionGoogleHybrid         = "y";`)
From what I understand in the comments, I think "Hybrid" was a layer that needed to be overlay on Satellite layer to be consistent. If by adding this "y" to the url request give us a complete layer that's great !

**Comment by tilaktilak** [comment]
Also @ans10528 can you confirm that caching Google Hybrid with the OfflineMaps menu and import them again works fine ?

**Comment by ans10528** [comment]
Google Hybrid export and import is works
https://youtu.be/RmMcJczWLwA

**Comment by ans10528** [comment]
> ans10528 add a new argument that was missing in the URL (`_versionGoogleHybrid = "y";`)
> From what I understand in the comments, I think "Hybrid" was a layer that needed to be overlay on Satellite layer to be consistent. If by adding this "y" to the url request give us a complete layer that's great !

http://mt1.google.com/vt/lyrs=s&x=1325&y=3143&z=13
lyrs=s x=1325 y=3143 z=13
+
http://mt1.google.com/vt/lyrs=h&x=1325&y=3143&z=13
lyrs=h x=1325 y=3143 z=13
=
http://mt1.google.com/vt/lyrs=y&x=1325&y=3143&z=13
lyrs=y x=1325 y=3143 z=13


https://stackoverflow.com/questions/23017766/google-maps-tile-url-for-hybrid-maptype-tiles
h = roads only
m = standard roadmap
p = terrain
r = somehow altered roadmap
s = satellite only
t = terrain only
y = hybrid

sorry i'm not good at english

**Pull request merged**